.scala-steward.conf

```diff
@@ -1,7 +1,4 @@
-pullRequests.frequency = "@monthly"
-
 updates.ignore = [
-  { groupId = "com.google.protobuf", artifactId = "protobuf-java" }
```

Comment:
I think we should ignore protobuf-java. Newer versions have breaking changes.

Reply:
Done, I added it back in but in a better spot